install/operator/pkg/generator/assets/infrastructure/03-syndesis-server-config.yml.tmpl

```diff
@@ -78,6 +78,8 @@
           {{- else}}
           enabled: false
           {{- end}}
+        auditing:
+          enabled: {{.Syndesis.Components.Server.Features.Auditing}}
 {{- if .Syndesis.Addons.Jaeger.Enabled}}
```

Comment:
This line seems to be in the wrong place ??
Does `dblogging` have anything to do with jaeger?

@claudio4j

Reply:
Yeah, we have two implementations JsonDB and Jaeger (https://github.com/syndesisio/syndesis/tree/1.12.x/app/server/logging), so when Jaeger is turned on the JsonDB one is turned off. (but @claudio4j would know better)

Reply:
+1 @zregvart.